tests/algo/test_cfr.py

```diff
@@ -3,12 +3,11 @@
 import time
 
 import unittest
+from buffalo import CFR, CFROption
+from buffalo import aux, set_log_level
+from buffalo import StreamOptions
```

Comment:
import 순서 아래처럼 수정하면 좋을거 같습니다.

```suggestion
import time
import unittest

from buffalo import CFR, CFROption
from buffalo import aux, set_log_level
from buffalo import StreamOptions
```

Reply:
수정했습니다!